Examples/Scripts/Python/material_mapping.py

```diff
@@ -139,4 +162,6 @@ def runMaterialMapping(
         outputDir=os.getcwd(),
         inputDir=os.getcwd(),
         readCachedSurfaceInformation=False,
+        mapName=mapName,
+        mapFormat=fmt
```

Comment:
```suggestion
        mapName=mapName,
        mapFormat=mapFormat
```
Maybe, it makes sense to change from `fmt` to `mapFormat` to follow the overall style and be a bit more explicit.